diff --git a/gulpfile.js b/gulpfile.js
index 4c2d5a06..16a7fef7 100644
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -264,6 +264,7 @@ gulp.task('apps', ['dist', 'clean-apps'], function (done) {
         files: './dist/**/*',
         buildDir: appsDir,
         platforms: platforms,
+        version: '0.27.0',
         flavor: 'normal',
         macIcns: './images/bf_icon.icns',
         macPlist: { 'CFBundleDisplayName': 'Betaflight Configurator'},
